main-app/app/scripts/providers/proxy/proxy.js

```diff
@@ -0,0 +1,24 @@
+(function () {
+    'use strict';
+    angular.module('Tombola.NoughtsAndCrosses.Providers.Proxy')
+        .service('Proxy',['$http','ProxyConstants', function($http, ProxyConstants) {
+            var Proxy = function() {
+                var constant = ProxyConstants;
+                var me = this;
+
+                me.setupBoard = function (msg) {
+                    return post(constant.newGameURL, msg, {withCredentials: true});
+                };
+
+                me.makeMove = function (msg) {
+                    return post(constant.moveURL, msg, {withCredentials: true});
```

Comment:
Small thing, but put the credentials in the "post" function - always need it & no need to repeat.